tools/run-dev.py

```diff
@@ -305,7 +305,6 @@ class Application(web.Application):
     def __init__(self):
         # type: () -> None
         handlers = [
-            (r"/sockjs/.*/websocket$", TornadoHandler),
```

Comment:
yeah I've used different handlers for websocket and http at the begining, but later I've made combine handler which can handle both connections by headers. It works with this route, but it's unnecessary and route 'sockjs.*'  can handle both connection types.